initial-promises/node-server/promises.cf

```diff
@@ -437,6 +444,11 @@ bundle agent garbage_collection
         file_select => days_old("30"),
         depth_search => recurse("inf");
 
+      "${log_files_to_rotate}"
+        comment     => "Rotate file if above specified size",
+        rename      => rotate("10"),
+        file_select => bigger_than("1M");
```

Comment:
/var/rudder/cfengine-community//bin/cf-agent -KIb garbage_collection 
2016-05-20T12:35:37+0200     info: Using command line specified bundlesequence
2016-05-20T12:35:37+0200     info: /default/garbage_collection/files/'/var/rudder/cfengine-community/outputs'[0]: Deleted file '/var/rudder/cfengine-community/outputs/cf_orchestrateur_1_labo_normation_com__1463135408_Fri_May_13_12_30_08_2016_0x7fcb3ef97700'
2016-05-20T12:35:47+0200     info: /default/garbage_collection/files/'/var/rudder/cfengine-community/promise_summary.log'[1]: Rotating files '/var/rudder/cfengine-community/promise_summary.log' in 10 fifo